frontend/components/commons/forms/questions-form/fields/multi-label/MultiLabelSelection.component.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <div class="container">
+    <div class="component-header" v-if="showSearch || showCollapseButton">
```

Comment:
What happens if the `div` section is always rendered? Will affect how the search bar, the collapse button, and the labels will be shown?

Reply:
The purpose of the conditional rendering is to not have empty `div` in the dom if the question does not have a `search` or a `collapse button` . 

Without the div it will be like this : 
![image](https://github.com/argilla-io/argilla/assets/88380932/a73d43b5-d890-4953-ac80-65dd26c2013c)